src/lightweight_helper.js

```diff
@@ -0,0 +1,31 @@
+
+/**
+ * @param {String} path
+ * @param {String} integrity
+ */
+export const loadScript = (path, integrity) => {
+  // eslint-disable-next-line
+    if(self.importScripts) {
```

Comment:
```suggestion
  if (typeof importScripts !== 'undefined') {
```
Maybe then you can also remove the `eslint-disable-next-line`?

Reply:
fixed